ui/src/pages/Feed.svelte

```diff
@@ -118,35 +119,16 @@
     sortedRecommendations = getCollectedItemLeaderboard(me).slice(0, 4);
   });
 
-  const handlePost = ({ detail: { content, uploadedImageUrl, ref } }) => {
-    let post = {};
-    console.log({ ref });
-    if (ref) {
-      // Here we need to create the retweet post instead of the type "other"
-      post = {
-        ...post,
-        bespoke: {
-          retweet: {
-            ref: ref,
-            blurb: content || '',
-          },
+  const handlePost = ({ detail: { content, uploadedImageUrl } }) => {
+    api.portal.do.create({
+      bespoke: {
+        other: {
+          title: '',
+          blurb: content || '',
+          link: '',
+          image: uploadedImageUrl || '',
         },
-      };
-    } else {
-      post = {
-        ...post,
-        bespoke: {
-          other: {
-            title: '',
-            blurb: content || '',
-            link: '',
-            image: uploadedImageUrl || '',
-          },
-        },
-      };
-    }
-    post = {
-      ...post,
+      },
```

Comment:
These changes revert the changes which are required to quote chat messages, I think it's git's fault, but please revise